spec/reprocess_spec.rb

```diff
@@ -0,0 +1,65 @@
+require File.expand_path('../spec_helper', __FILE__)
+require 'energy_globe_awards/reprocess'
+
+describe 'Reprocess' do
+  before do
+    @source = File.expand_path(
+      '../fixtures/awards.json', __FILE__)
+    @dest = File.expand_path(
+      '../fixtures/awards-reprocessed.json', __FILE__)
+    @processor = Reprocess.new(@source, @dest)
+    @processor.reprocess
+  end
+
+  it 'reverses the list' do
+    raw_source = File.read(@source)
+    raw_dest = File.read(@dest)
+    source_list = JSON.load(raw_source)
+    dest_list = JSON.load(raw_dest)
+    # because hyphens
+    dest_list.first['title'][0..3].must_equal(
+      source_list.reverse.first['title'][0..3])
+  end
+
+  it 'breaks the json onto lines' do
+    raw = File.read(@dest)
+    raw.split("\n").size.must_equal 6
+  end
+
+  it "removes descriptions" do
+
+  end
+
+  it 'adds hyphens to titles' do
+    raw_dest = File.read(@dest)
+    dest_list = JSON.load(raw_dest)
+    title = dest_list.last['title']
+    title.must_match(/&shy;/)
+  end
+
+  it 'adds hyphens to orgs' do
+    raw_dest = File.read(@dest)
+    dest_list = JSON.load(raw_dest)
```

Comment:
i'm not going to refactor my tests until i am sure i am done with them.
there isn't enough overhead here. This so very low priority.